bin/pycbc_compress_bank

```diff
@@ -40,57 +41,66 @@ from pycbc import filter
-                     "will be used in the compressed waveforms. Default is "
-                     "0.001.")
+                    help="The minimum duration used for t(f) in seconds. "
+                    "The inverse of this gives the maximum frequency "
+                    "step that will be used in the compressed waveforms. "
+                    "Default is 0.001.")
 parser.add_argument("--tolerance", type=float, default=0.001,
-                help="The maximum mismatch to allow between the interpolated "
-                     "waveform must and the full waveform. Points will be "
-                     "added to the compressed waveform until its "
-                     "interpolation has a mismatch <= this value. Default is "
-                     "0.001.")
+                    help="The maximum mismatch to allow between the "
+                    "interpolated waveform must and the full waveform. "
+                    "Points will be added to the compressed waveform "
+                    "until its interpolation has a mismatch <= this value. "
+                    "Default is 0.001.")
 parser.add_argument("--interpolation", type=str, default="linear",
-                help="The interpolation to use for decompressing the "
-                     "waveforms for checking tolerance. Options are 'linear', "
-                     "or any interpolation recognized by scipy's interp1d "
-                     "kind argument. Default is linear.")
+                    help="The interpolation to use for decompressing the "
+                    "waveforms for checking tolerance. Options are "
+                    "'linear', or any interpolation recognized by scipy's "
+                    "interp1d kind argument. Default is linear.")
 parser.add_argument("--precision", type=str, choices=["double", "single"],
-                default="double",
-                help="What precision to generate and store the waveforms with;"
-                     " default is double.")
+                    default="double",
+                    help="What precision to generate and store the "
+                    "waveforms with; default is double.")
 parser.add_argument("--force", action="store_true", default=False,
                     help="Overwrite the given hdf file if it exists. "
-                         "Otherwise, an error is raised.")
+                    "Otherwise, an error is raised.")
 parser.add_argument("--verbose", action="store_true", default=False)
 
+# Insert the PSD options
+pycbc.psd.insert_psd_option_group(parser, include_data_options=False)
 
 args = parser.parse_args()
 
 pycbc.init_logging(args.verbose)
 
+# Verify if the input options for calculating the psd are correct, if
+# a PSD is to be used while calculating the overlaps in compress.py . If
+# using a PSD is not desired, this check would prevent raising an error
+# asking for PSD options.
+if args.psd_model or args.psd_file or args.asd_file or args.psd_estimation :
```

Comment:
`verify_psd_options` complains if none of `args.psd_model` or `args.psd_file` or `args.asd_file` is provided, for example :
```(pycbc-dev-sugwg012417) [soumi.de@sugwg-condor bin]$ pycbc_compress_bank --bank-file H1L1-BANK2HDF-1128299417-1083600.hdf --output test_psd.hdf --low-frequency-cutoff 30 --approximant 'TaylorF2:mtotal<4' 'SEOBNRv2_ROM_DoubleSpin:else' --sample-rate 4096  --tmplt-index 3 5 --compression-algorithm spa --precision single  --verbose --force

pycbc_compress_bank: error: you must supply one of the following --psd-file, --psd-model, --psd-estimation, --asd-file
```